### PR TITLE
add hpc category

### PR DESCRIPTION
**Pull request by fffrog**
add hpc category

### CODE REVIEW

File: _includes/projects.html [hunk 1]
```diff
@@ -206,4 +206,27 @@ <h5 class="font-weight-bold text-white bg-purple mb-0 text-center">{{ item.proje
             </div>
         </div>
     </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-12 col-sm-2 text-center bg-blue text-white d-flex flex-column justify-content-center align-items-center">
+                <h4 class="">OpenHPC</h4>
```
Comment (martin-g) [inline _includes/projects.html:212]: Shouldn't this be `HPC` ?
Many of the projects in `category: hpc` below are not components in OpenHPC.

File: _posts/2021-08-31-CP2K.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (martin-g) [inline _posts/2021-08-31-CP2K.md:1]: Does the date in the filename matter ?

File: _posts/2021-08-31-Tez.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+meta:
+  keywords: "Apache Tez"
+
+project:
+  title: "Apache Tez"
+  type: "Java"
+  url: "https://github.com/apache/tez"
+  project_official_url: "https://tez.apache.org/"
+  logo: "/assets/images/projects/libraries/ant/logo.png"
```
Comment (martin-g) [inline _posts/2021-08-31-Tez.md:14]:
```suggestion
  logo: "/assets/images/projects/libraries/tez/logo.png"
```

**Review by ZhengZhenyu — APPROVED** [review]
lgtm

File: _posts/2021-08-31-Tez.md (hunk 3 above)
Comment (fffrog) [reply _posts/2021-08-31-Tez.md:14]: Sorry for late.
I will submit a new pr to fix it.
Thank you.

**Pull request merged**